store/bolt/store.go

```diff
@@ -0,0 +1,75 @@
+package bolt
+
+import (
+	"encoding/binary"
+	"os"
+	"path/filepath"
+	"time"
+
+	"github.com/boltdb/bolt"
+)
+
+//go:generate protoc --gogo_out=. internal/internal.proto
+
+// Store represents the BoltDB-backed data store.
+type Store struct {
+	path string
+	db   *bolt.DB
+
+	*UserStore
+}
```

Comment:
The `Store` holds the Bolt database reference and it's a container for all the substores (e.g. `UserStore`). It embeds the substores (which means it has them as unnamed members) so that `Store` can be used as a `UserStore` in code. It's a nice way to have `Store` allow us to access everything without attaching a ton of methods directly to it.